pyp5js/tests/http/test_web_app.py

```diff
@@ -0,0 +1,108 @@
+import os
+import shutil
+from pyp5js.config import SKETCHBOOK_DIR
+from pyp5js.http.web_app import app as web_app
+from flask_testing import TestCase
+
+class Pyp5jsWebTestCase(TestCase):
+    render_templates = False
+
+    def create_app(self):
+        app = web_app
+        app.config['TESTING'] = True
+        return app
+
+    def setUp(self):
+        if not SKETCHBOOK_DIR.exists():
+            SKETCHBOOK_DIR.mkdir()
+
+    def tearDown(self):
+        if SKETCHBOOK_DIR.exists():
+            shutil.rmtree(SKETCHBOOK_DIR)
+
+    def create_sketch(self, name):
+        sketch_dir = SKETCHBOOK_DIR.joinpath(name)
+        os.makedirs(sketch_dir)
+        target_file = sketch_dir.joinpath(f'{name}.py')
+        index_file = sketch_dir.joinpath('index.html')
+        self.create_file(target_file)
+        self.create_file(index_file, 'index file content')
+        return sketch_dir
+
+    def create_sketch_with_static_files(self, name, static_path):
+        sketch_dir = SKETCHBOOK_DIR.joinpath(name)
+        static_dir = sketch_dir.joinpath(static_path.split('/')[0])
+        os.makedirs(static_dir)
+        target_file = sketch_dir.joinpath(f'{name}.py')
+        index_file = sketch_dir.joinpath('index.html')
+        static_file = static_dir.joinpath(static_path.split('/')[1])
+        self.create_file(target_file)
+        self.create_file(index_file)
+        self.create_file(static_file, 'static file content')
+        return sketch_dir
```

Comment:
Same refactoring tip here but now combining the `new` command with the `transcrypt`. The second one is responsible for creating the static files and all the other dependencies.

```python
def create_sketch_with_static_files(self, name):
    sketch_files = commands.new(name)
    commands.transcrypt(name)
    return sketch_fiiles.sketch_dir
```
```